src/QMCDrivers/tests/test_QMCCostFunctionBase.cpp

```diff
@@ -55,7 +56,8 @@ TEST_CASE("updateXmlNodes", "[drivers]")
   const SimulationCell simulation_cell;
   MCWalkerConfiguration w(simulation_cell);
   QMCHamiltonian h;
-  TrialWaveFunction psi;
+  ProjectData project_data;
```

Comment:
Can we just use RuntimeOptions instead of ProjectData? Maybe also put RuntimeOptions in a separate header file.

Reply:
ProjectData objects currently aligns with the current build picking the appropriate RuntimeOptions for complex/real or full/mixed precision. I might have to move this at the RuntimeOptions constructor level.